packages/blocks/table/src/Table.tsx

```diff
@@ -226,7 +226,10 @@ export const Table: BlockComponent<AppProps> = ({
             data: results as TableData["data"],
             linkedAggregation: {
               ...tableData.linkedAggregation,
-              operation,
+              operation: {
+                ...linkedData.operation,
+                ...operation,
+              },
```

Comment:
Previously the table block was completely replacing the `operation` stored in its state with whatever the `aggregateEntity` external returned as the new `operation`. This caused the Table to crash previously, because the `operation` returned by `aggregateEntity` did not include `entityTypeId` which the table block currently relies on. This would prevent a similar crash from happening in an embedding application environment where `aggregateEntity` returns less fields for the `operation` than required by the Table block.